receiver/awscontainerinsightreceiver/internal/prometheusscraper/decoratorconsumer/decorator_test.go

```diff
@@ -62,80 +62,80 @@ func TestConsumeMetrics(t *testing.T) {
 			ShouldError: false,
 		},
 		"unit": {
-			Metrics: GenerateMetrics(map[MetricIdentifier]map[string]string{
-				{util, pmetric.MetricTypeGauge}: {
+			Metrics: GenerateMetrics(map[MetricIdentifier][]map[string]string{
+				{util, pmetric.MetricTypeGauge}: {{
```

Comment:
this is failing lint check https://github.com/amazon-contributing/opentelemetry-collector-contrib/actions/runs/8421014667/job/23056990611?pr=187